index.js

```diff
@@ -70,7 +70,7 @@ function fastifyCaching (instance, options, next) {
       value += `, s-maxage=${_options.serverExpiresIn}`
     }
 
-    instance.addHook('onRequest', (req, res, next) => {
+    instance.addHook('onRequest', function cachingSetCacheControlHeader (req, res, next) {
```

Comment:
This is really the only change needed in this PR. The other names are fine; the fact that they are named is enough. And we don't _need_ names in the tests. The names provide information when reviewing stack traces, stepping through with a debugger, and reviewing observability data.